validation-test/stdlib/CollectionType.swift.gyb

```diff
@@ -8,6 +8,9 @@
 // <rdar://problem/24357067> The compiler never finishes compiling validation-test/stdlib/CollectionType.swift.gyb in -O
 // REQUIRES: swift_test_mode_optimize_none
 
+// FIXME: This test takes too long to complete on Android.
+// UNSUPPORTED: OS=linux-androideabi
```

Comment:
Could you check again?  We have split this test recently.  It takes less than 1 second OS X.